renpy/test/testfocus.py

```diff
@@ -76,6 +76,7 @@ def match(f):
 
 def relative_position(x, posx, width):
     if posx is not None:
+        # TODO : check this regarding the new absolute.compute
```

Comment:
This has little to do with the rest, as it's not about transform or style properties. I'd advocate converting it to the new system, but it's really a separate issue.